README.md

```diff
@@ -1,9 +1,9 @@
-
-
 # cpp-extra-samples
 
 This repository contains additional **C++** code samples for **Zivid**.
 
+[![Build Status](https://dev.azure.com/zivid-devops/cpp-extra-samples/_apis/build/status/zivid.cpp-extra-samples?branchName=master)](https://dev.azure.com/zivid-devops/cpp-extra-samples/_build/latest?definitionId=4&branchName=master)
```

Comment:
We use [shields.io](https://shields.io) for badges in all repositories. Then we get clean, nice and consistent UI across repos, and also consistency when we have multiple badges in same repo ([example](https://github.com/zivid/zivid-python#zivid-python)).

Reply:
Also use the footnote style used in zivid-python, it makes the readme more readable.

Reply:
I did refer to that repo when setting this up, but since it's broken I didn't use it:

![Screenshot from 2019-08-19 09-33-30](https://user-images.githubusercontent.com/727543/63247070-7069a280-c264-11e9-97a6-782adc2c1113.png)

When that gets fixed, I'll update this one too. In the mean time I suggest merging the current PR so we at least have something showing the build status.